JavaScript/JavaScriptSDK.Interfaces/Contracts/Generated/RemoteDependencyData.ts

```diff
@@ -1,51 +1,99 @@
-// THIS TYPE WAS AUTOGENERATED
+// THIS FILE WAS AUTOGENERATED
 /// <reference path="Domain.ts" />
 /// <reference path="DataPointType.ts" />
 /// <reference path="DependencyKind.ts" />
 /// <reference path="DependencySourceType.ts" />
 module AI
 {
 "use strict";
+    
+    /**
+     * An instance of Remote Dependency represents an interaction of the monitored component with a remote component/service like SQL or an HTTP endpoint.
+     */
     export class RemoteDependencyData extends Microsoft.Telemetry.Domain
     {
+        
+        /**
+         * Schema version
+         */
         public ver: number;
+        
+        /**
+         * Name of the command initiated with this dependency call. Low cardinality value. Examples are stored procedure name and URL path template.
+         */
         public name: string;
+        
+        /**
+         * Identifier of a dependency call instance. Used for correlation with the request telemetry item corresponding to this dependency call.
+         */
         public id: string;
+        
+        /**
+         * Result code of a dependency call. Examples are SQL error code and HTTP status code.
+         */
         public resultCode: string;
-        public duration: string;
-        public success: boolean;
-        public data: string;
-        public target: string;
-        public type: string;
-        public properties: any;
-        public measurements: any;
-
-        /* Following fields should be marked as depricated in the next release, and must be removed in the subsequent breaking release */
         public kind: AI.DataPointType;
         public value: number;
+        
+        /**
+         * Request duration in format: DD.HH:MM:SS.MMMMMM. Must be less than 1000 days.
+         */
+        public duration: string;
         public count: number;
```

Comment:
these fields gone in public schema

Reply:
did we replace duration with 'value'? Or it's just gone and the JS SDK should not report any duration?

Reply:
it's there: https://github.com/Microsoft/ApplicationInsights-Home/blob/master/EndpointSpecs/Schemas/Bond/RemoteDependencyData.bond

Reply:
I meant `count`. It's not there

Reply:
Yes, I reviewed all bond files in AppInsights-Home and remove all fields that are not present there. 

After removing those fields I've noticed some no longer used code, which I'm removing right now.